contracts/token/ERC7984/extensions/ERC7984Rwa.sol

```diff
@@ -0,0 +1,181 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.27;
+
+import {FHE, externalEuint64, euint64} from "@fhevm/solidity/lib/FHE.sol";
+import {AccessControl} from "@openzeppelin/contracts/access/AccessControl.sol";
+import {ERC165} from "@openzeppelin/contracts/utils/introspection/ERC165.sol";
+import {Multicall} from "@openzeppelin/contracts/utils/Multicall.sol";
+import {Pausable} from "@openzeppelin/contracts/utils/Pausable.sol";
+import {IERC7984} from "./../../../interfaces/IERC7984.sol";
+import {IERC7984RwaBase} from "./../../../interfaces/IERC7984Rwa.sol";
```

Comment:
Maybe it'd be cleaner to use absolute paths vs relative? I don't think we have any guidelines against it.

Reply:
Seems consistent with what we already have, I don't see absolute paths for project files across our libs, do you?